photoeditor/src/test/java/ja/burhanrashid52/photoeditor/EnumTest.kt

```diff
@@ -11,11 +11,6 @@ class EnumTest {
         assertEquals(ViewType.values().size.toLong(), 4)
     }
 
-    @Test
-    fun testNumberOfShapeTypes() {
```

Comment:
Is there alternative way to check how may shapes we support ?

Reply:
In new Kotlin version (which I plan to update after #497) `when` expression must be exhaustive. That means if some shape will be added or removed, there will be compilation error in `DrawingView:createShape():141` with message `'when' expression must be exhaustive, add necessary 'NewShapeName' branch or 'else' branch instead` or in case of shape removal: `Unresolved reference: Brush`.

There is a possibility to obtain how many elements has sealed class but we need to use a reflection for this. To use reflection in Kotlin we need `org.jetbrains.kotlin:kotlin-reflect` dependency.

For this reasons I think this test is no longer needed and not worth the effort of adding new dependency only for this.